Feature bread profiles page

MalasataXD opened the pull request:
## STATUS
There was been done A LOT of improvement to other things and the BreadProfile Page...

## CHANGES
There should be no need for any changes.

BjarkeBP committed:
Bread profile page

*Added
-BreadProfilesPage setting comp to together and making states in the page
to control components

*Changes
-needs css and testing :D

BjarkeBP committed:
BreadProfilesPage error text

*Added
-Error text, so the user can se the error

MalasataXD committed:
Changes to SelectedBreadProfile

* ADDED
- New CSS
- Comments to the code

* CHANGED
- Moved BreadProfile and target interface to config

BjarkeBP committed:
Merge branch 'dev' of https://github.com/SEP4-DK1/SEP4-FRONTEND-TS into feature-breadProfilesPage

BjarkeBP committed:
Path fix

*fixed
-SelcetedBreadProfiles -> SelectedBreadProfile

MalasataXD committed:
MANY MANY CHANGES!

* ADDED
- New CSS for TargetCard
- BreadProfile link to Router

* CHANGED
- Reformatted THE CODE + CSS for SelectedBreadProfile
- Reformatted the code for TargetCard
- Fixed naming in some of the interfaces...
- Made a system for the things in config.ts

Grey-rgb reviewed (dismissed):
Immaculate 
@BjarkeBP Is the G.O.A.T

MalasataXD committed:
Removed Tests

We can't this the TargetCard component, since the functionality of the code has changed. And the checks are now performed on another component.

Grey-rgb reviewed (approved):
Yes yes
Very nice 🔥 💀 🤡 👍 💯 

🛏️ 😴
